kafka-rest/src/main/java/io/confluent/kafkarest/response/JsonStreamMessageBodyReader.java

```diff
@@ -41,7 +42,9 @@ public JsonStreamMessageBodyReader(ObjectMapper objectMapper) {
   @Override
   public boolean isReadable(
       Class<?> type, Type genericType, Annotation[] annotations, MediaType mediaType) {
-    return JsonStream.class.equals(type) && MediaType.APPLICATION_JSON_TYPE.equals(mediaType);
+    return JsonStream.class.equals(type)
+        && MediaType.APPLICATION_JSON_TYPE.getType().equalsIgnoreCase(mediaType.getType())
+        && MediaType.APPLICATION_JSON_TYPE.getSubtype().equalsIgnoreCase(mediaType.getSubtype());
```

Comment:
Not sure if this is correct but I think the third condition should be optional if there is subtype specified, something like
`&& (mediaType.getSubtype() == null || MediaType.APPLICATION_JSON_TYPE.getSubtype().equalsIgnoreCase(mediaType.getSubtype()))`